invenio_jobs/assets/semantic-ui/js/invenio_jobs/administration/api.js

```diff
@@ -0,0 +1,26 @@
+// This file is part of Invenio
+// Copyright (C) 2024 CERN.
+//
+// Invenio RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { http } from "react-invenio-forms";
+import React from "react";
+
+import { ErrorMessage } from "@js/invenio_administration";
+
+export const stopRun = async (url) => {
```

Comment:
is this being used? I can't find any reference to this. 

If not, we can remove the file.

Reply:
yeah this was when I was doing something a bit more generic but I've put the API call inside the buttons now